docs/source/lda.rst

```diff
@@ -94,4 +94,15 @@ The package provides several functions to perform Linear Discriminant Analysis.
 
     :return: The resultant linear discriminant functional of type ``LinearDiscriminant``.
 
-
+    **Keyword arguments:**
+
+    ============== =============================================================== =========================
+      name                    description                                                          default
+    ============== =============================================================== =========================
+     covestimator   custom covariance estimator; when it is different from           ``SimpleCovariance()``
+                    ``nothing``, the covariance matrix will be calculated as
```

Comment:
`nothing` isn't accepted AFAICT since we now use `SimpleCovariance()`. Also better restrict the type of the argument in all signatures.

Reply:
Sorry for overlooking this error in documentation. I've fixed it and restricted the type of `covestimator` in all relevant functions.